### PR TITLE
Bigger add desktop modal and new cleaner selected styles

### PR DESCRIPTION
**Pull request by jvinolas**
This will bring a bigger and more cleaner add desktop modal
imatge

### CODE REVIEW

**Review by NefixEstrada — APPROVED** [review]

File: webapp/webapp/webapp/static/js/desktops.js [hunk 1]
```diff
@@ -41,9 +43,8 @@ $(document).ready(function() {
                     opacity: 1,
                     type: 'error'
                 });                
-        }else{	
+        }else{
```
Comment (sim6) [inline webapp/webapp/webapp/static/js/desktops.js:46]:
```suggestion
        }else{
```

**Review by sim6 — CHANGES_REQUESTED** [review]
Please, could you remove tailing whitespaces?

**Review by sim6 — CHANGES_REQUESTED** [review]
Thanks! Please, Could you squash the commits?

**Review by sim6 — APPROVED** [review]
Thanks!

**Pull request merged**